frontend/components/Header/Nav/Links/SupportTheGuardian.tsx

```diff
@@ -40,8 +42,38 @@ const SupportTheGuardian: React.SFC<{
     children: React.ReactChild;
     href: string;
 }> = ({ className, children, href, ...props }) => (
-    <a className={cx(style, className)} href={href} {...props}>
-        {children}
-    </a>
+    <AsyncClientComponent f={shouldShow}>
+        {({ data }) => (
+            <>
+                {data && (
+                    <a className={cx(style, className)} href={href} {...props}>
+                        {children}
+                    </a>
+                )}
+            </>
+        )}
+    </AsyncClientComponent>
 );
+
+const shouldShow: () => Promise<boolean> = () =>
```

Comment:
You could just return `Promise.resolve(!(isRecentContributor() || isPayingMember()));` here;